### PR TITLE
Sorr task612 get information from signal

### PR DESCRIPTION
**Pull request by why2000**
https://docs.google.com/document/d/1z-svt4K6dSy5ncuVZzrtL9ZYuBmknCp8UfGfxljaBNQ/edit#heading=h.g37b603qcnlr
#612 
FYI @samarth9008 @gpsaggese 
This PR is using the structure from #609 and based on it. Better to review after merging that PR.

### CODE REVIEW

File: marketing/signal/extract_investors_from_signal_list.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
+"""
+Import as:
+
+import marketing.signal.extract_investors_from_signal_list as mseifsili
+"""
+
+# !sudo /bin/bash -c "(source /venv/bin/activate; pip install --upgrade selenium webdriver-manager)"
```
Comment (samarth9008) [inline marketing/signal/extract_investors_from_signal_list.py:7]: Remove it if not used.

File: marketing/notebooks/SorrTask612_Get_information_from_Signal.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# %%
+import marketing.signal.extract_investors_from_signal_list as mseifsili
+
+# %% run_control={"marked": true}
+# Destination result file path.
+investors_csv_save_path = "../result_csv/signal_fintech_investors.csv"
+# Source data page URL.
+baseurl = "https://signal.nfx.com/investor-lists/top-fintech-seed-investors"
+start_idx = 0
+length = 40
```
Comment (samarth9008) [inline marketing/notebooks/SorrTask612_Get_information_from_Signal.py:28]: Can you explain why are these required?

File: marketing/signal/extract_investors_from_signal_list.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    baseurl: str, start_idx: int, length: int
+) -> pd.DataFrame:
+    """
+    Extract a dataframe of investor information from a signal investors list
+    page. e.g. https://signal.nfx.com/investor-lists/top-fintech-seed-investors
+    Available lists are in this page: https://signal.nfx.com/investor-lists/
+
+    :param baseurl: str, the page url to be extracted
+    :param start_idx: int, the index of the first item to be extracted (start from 0)
+    :param length: int, the number of items to be extracted
```
Comment (samarth9008) [inline marketing/signal/extract_investors_from_signal_list.py:30]: No types in param description.

**Review by samarth9008 — COMMENTED** [review]
I see there are errors in your notebook. Is there any reason for it?

File: marketing/signal/extract_investors_from_signal_list.py (hunk 1 above)
Comment (why2000) [reply marketing/signal/extract_investors_from_signal_list.py:7]: The error in notebook is because I'm using WSL and selenium cannot open the browser under WSL jupyter notebook. I've tested this on an macbook and it works fine. For this line: We'll need to install those dependencies to get the code running, since selenium is not included in the image.

File: marketing/notebooks/SorrTask612_Get_information_from_Signal.py (hunk 2 above)
Comment (why2000) [reply marketing/notebooks/SorrTask612_Get_information_from_Signal.py:28]: This is because the page is not loading all items at first. I'm not sure of the total length of it, so I want to give a range of data to load, thus it won't have a too long waiting time.

**Review by gpsaggese — COMMENTED** [review]
Henry is officially the scraping ninja on the team.
I can't add much, besides let's make sure to document things both in code and outside.

I'll let @samarth9008 work with Henry on this stuff.

File: marketing/signal/extract_investors_from_signal_list.py (hunk 1 above)
Comment (samarth9008) [reply marketing/signal/extract_investors_from_signal_list.py:7]: A better thing would be to add a comment above it regarding its usage since this is not used anywhere in this file but will be used in another file which will make use of this library.

File: marketing/notebooks/SorrTask612_Get_information_from_Signal.py (hunk 2 above)
Comment (samarth9008) [reply marketing/notebooks/SorrTask612_Get_information_from_Signal.py:28]: Please mention this in the comment.

File: marketing/notebooks/SorrTask601_Extract_VCs_from_Tra_search_mhtml.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# %% run_control={"marked": true}
+import marketing.tra.extract_VCs_from_Tra_search_mhtml as mtevftsmh
+
+# %% [markdown]
+# # Workflow before using this library:
```
Comment (gpsaggese) [inline marketing/notebooks/SorrTask601_Extract_VCs_from_Tra_search_mhtml.py:19]: Good explanation.

Can we move to our new approach to doc following diataxis.fr ?
 
Like an `marketing.explanation.md` and `tra.how_to_guide.md`?
@samarth9008 can you help? We can do a PR with the outline of the doc and then fill it up

File: marketing/signal/extract_investors_from_signal_list.py [hunk 5]
```diff
@@ -0,0 +1,102 @@
+"""
+Import as:
+
+import marketing.signal.extract_investors_from_signal_list as mseifsili
+"""
+
+# TODO(Henry): This package need to be manually installed until they are added
```
Comment (gpsaggese) [inline marketing/signal/extract_investors_from_signal_list.py:7]: it is added

just to show that I do read the code :-)

**Review by gpsaggese — COMMENTED** [review]
Good work. Let's merge and then we can do a follow up.

File: marketing/notebooks/SorrTask601_Extract_VCs_from_Tra_search_mhtml.py (hunk 4 above)
Comment (samarth9008) [reply marketing/notebooks/SorrTask601_Extract_VCs_from_Tra_search_mhtml.py:19]: @why2000 

Instead of writing this in a notebook, we would like to create a md doc like `tra.how_to_guide.md`, explaining its workflow end-to-end. Toe location will be `marketting/tra`

Same for signal and for @yiyunlei same for `dropcontact` if applicable. Location will be there respective folders.

File: marketing/signal/extract_investors_from_signal_list.py (hunk 5 above)
Comment (why2000) [reply marketing/signal/extract_investors_from_signal_list.py:7]: Which version of images are they added? I still cannot import selenium on dev server's notebook yet. Also `webdriver-manager` needs to be added together for selenium to work: we don't want to download webdriver manually on server. It won't bundle any driver binary inside, but to auto-determine which version of driver to be downloaded when we start selenium.
